.travis.yml

```diff
@@ -27,14 +27,17 @@ before_install:
 
 matrix:
   fast_finish: true
-# Allow nightly to fail
+  # Allow nightly to fail
   allow_failures:
     - rust: nightly
     - rust: beta # until ipfs-api is un-regressed
   # Only check formatting when running against stable
   include:
     - env: CHECK_FORMATTING=true
-      if: group = stable
```

Comment:
If we do this in other releases, won't the format checks fail because other releases have other formatting rules?

Reply:
That option is not mandatory, by default it will be set to the first config of the list which is stable. I was having trouble doing `stable && tagged` logic and just removing the stable condition fixed it. I can try putting it back, this one in particular is trivial to put back, if you think it's more readable.

Reply:
We don't need to if it works.